isis/tests/Fixtures.cpp

```diff
@@ -256,6 +276,10 @@ namespace Isis {
   
 
   void MroCube::setInstrument(QString ikid, QString instrumentId, QString spacecraftName) {
+    // force real DNs
+    testCube->close();
+    testCube->setPixelType(PixelType::Real);
+    testCube->open(testCube->fileName(),"rw");
```

Comment:
@kaitlyndlee the magic sauce to force the pixel type to real, this got the tests for HiRise working again.